blocks/v2-feature-carousel/v2-feature-carousel.css

```diff
@@ -0,0 +1,76 @@
+:root {
+  --v2-feature-carousel-card-width: 150px;
+  --v2-feature-carousel-card-height: 200px;
+  --v2-feature-carousel-icon-size: 48px;
+  --v2-feature-carousel-svg-size: 24px;
+  --v2-feature-carousel-border-radius: var(--border-radius);
+}
+
+.v2-feature-carousel-container {
+  background-color: var(--c-grey-50);
+}
+
+.v2-feature-carousel__cards-list {
+  display: flex;
+  gap: 16px;
+  scroll-snap-type: x mandatory;
+  overflow: auto;
+  flex-flow: row nowrap;
+}
+
+.v2-feature-carousel__card {
+  display: flex;
+  flex: 0 0 auto;
+  width: var(--v2-feature-carousel-card-width);
+  height: var(--v2-feature-carousel-card-height);
+  padding: 24px;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: flex-start;
+  border-radius: var(--v2-feature-carousel-border-radius);
+  background: var(--c-white);
+  scroll-snap-align: center;
+}
+
+.v2-feature-carousel__card .icon {
+  width: var(--v2-feature-carousel-icon-size);
+  height: var( --v2-feature-carousel-icon-size);
+  border: 1px solid var(--c-grey-3);
+  border-radius: 50%;
+  justify-content: center;
+  align-items: center;
+}
+
+.v2-feature-carousel__card .icon svg {
+  width: var(--v2-feature-carousel-svg-size);
+  height: var(--v2-feature-carousel-svg-size);
+}
+
+.v2-feature-carousel__card .icon-graph svg {
+  fill: none;
+}
+
+@media screen and (min-width: 999px) {
```

Comment:
Please, don't use strange breakpoints. We decided on certain breakpoints, and we should adjust to them.
If you want to center the content you can also use to the first and last item of the list margin auto.

Reply:
done